index.html

```diff
@@ -14,11 +14,11 @@
         <header class ="page-header">
 
             <section class="frame-container">
-                <h2 id="logo-text" href="welcome">Welcome</h2>
+                <h2 id="logo-text">Welcome</h2>
```

Comment:
- [ ] As requested by the previous reviewer, kindly make sure to wrap your logo into an `a` tag which will not only increase our work to be more professional, it will even increase our website accessibility :wink:  You can always check [this](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

Reply:
The logo if rap up will demand a h tag by the section tag. This will alter the figma design. I commented on this while submitting